### PR TITLE
Icon specific tests

### PR DESCRIPTION
**Pull request by legaspielaine**
test for https://github.com/gambitph/Stackable/issues/1206

Tests passing.

### CODE REVIEW

File: cypress/integration/v2/blocks/icon.spec.js [hunk 1]
```diff
@@ -247,3 +248,41 @@ function advancedTab( viewport ) {
...
+		cy.toggleStyle( 'Title' )
+
+		// Set the block alignment to align center in toolbar
+		cy.adjustToolbar( 'Align', () => {
+			cy
+				.get( '.components-popover__content' )
+				.contains( 'Align center' )
+				.click( { force: true } )
+		}, {
+			buttonOrder: '0',
```
Comment (blankart) [inline cypress/integration/v2/blocks/icon.spec.js:269]: will it work if we change the parentSelector instead of adding `buttonOrder`?

Comment (legaspielaine) [reply cypress/integration/v2/blocks/icon.spec.js:269]: there are no unique classes in the parent div and the button itself, so that won't work

Comment (blankart) [reply cypress/integration/v2/blocks/icon.spec.js:269]: parentSelector should be able to handle this

Comment (legaspielaine) [reply cypress/integration/v2/blocks/icon.spec.js:269]: i can use :nth-of-type in the parent selector. Will commit 👍

**Review by blankart — APPROVED** [review]

**Pull request merged**